api/src/test/resources/io/strimzi/api/kafka/model/Kafka.out.yaml

```diff
@@ -17,13 +17,52 @@ spec:
       plain:
         authentication:
           type: "scram-sha-512"
+        networkPolicyPeers:
+        - namespaceSelector:
+            matchExpressions: []
+            matchLabels:
+              kafka-enabled: "true"
+          podSelector:
```

Comment:
is there a missing "-" here because `podSelector` is one more element in the `networkPolicyPeers` list? The same for all the other cases

Reply:
Actually, this is correct in the YAML file. The list consists of the `NetworkPolicPeer` objects. And this particular object combines the `podSelector` and `namespaceSelector`. It basically says _these pods from these namespaces can connect_: see https://kubernetes.io/docs/concepts/services-networking/network-policies/#behavior-of-to-and-from-selectors for more details.